backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/MultiplicityRangeImpl.java

```diff
@@ -67,7 +71,11 @@ protected EClass eStaticClass() {
      */
     @Override
     public EList<Expression> getBound() {
-        List<Usage> data = new ArrayList<>();
+        List<Expression> data = new ArrayList<>();
+        data.add(getLowerBound());
```

Comment:
Maybe you should check for not null? (Since line 104 you can return null)

Reply:
done